2024-12-11 20:18:52,856 - exceptions.py[line:34] - ERROR: Traceback (most recent call last):
  File "/home/HwHiAiUser/lingqi-xutao/gwproc/core/exceptions.py", line 18, in wrapper
    return func(*args, **kwargs)
  File "model_handler.py", line 69, in inference
    input_name = sess_hat.get_inputs()[0].name
AttributeError: 'AclLiteModel' object has no attribute 'get_inputs'

2024-12-12 10:00:03,500 - exceptions.py[line:34] - ERROR: Traceback (most recent call last):
  File "/data/theoWS/GoldWind/GW/gwproc/core/exceptions.py", line 18, in wrapper
    return func(*args, **kwargs)
  File "/data/theoWS/GoldWind/GW/gwproc/intrusion/model_handler.py", line 138, in inference
    sess = self.inference_sessions.get['intrusion']
TypeError: 'builtin_function_or_method' object is not subscriptable

2024-12-12 11:47:44,486 - exceptions.py[line:34] - ERROR: Traceback (most recent call last):
  File "/data/theoWS/GoldWind/GW/gwproc/core/exceptions.py", line 18, in wrapper
    return func(*args, **kwargs)
TypeError: BehaviorDetectWanderingHandler.inference() missing 1 required positional argument: 'data'

2024-12-12 11:49:11,077 - exceptions.py[line:34] - ERROR: Traceback (most recent call last):
  File "/data/theoWS/GoldWind/GW/gwproc/core/exceptions.py", line 18, in wrapper
    return func(*args, **kwargs)
  File "/data/theoWS/GoldWind/GW/gwproc/wandering/model_handler.py", line 873, in inference
    input_name = sess.get_inputs()[0].name
AttributeError: 'NoneType' object has no attribute 'get_inputs'

2024-12-12 14:22:56,789 - model_handler.py[line:39] - INFO: Model hat Loaded
2024-12-12 14:22:58,901 - model_handler.py[line:72] - INFO: Model hat Relased
2024-12-12 14:23:12,126 - model_handler.py[line:43] - INFO: Model intrusion Loaded
2024-12-12 14:23:20,534 - model_handler.py[line:74] - INFO: Model intrusion Relased
2024-12-12 14:23:49,736 - model_handler.py[line:793] - INFO: Model wandering Loaded
2024-12-12 14:23:52,414 - model_handler.py[line:825] - INFO: Model wandering Relased
2024-12-12 15:06:23,135 - model_handler.py[line:466] - INFO: Model cabinet_meter Loaded
2024-12-12 15:12:17,898 - model_handler.py[line:502] - INFO: Model cabinet_meter Relased
2024-12-12 15:14:52,542 - model_handler.py[line:466] - INFO: Model cabinet_meter Loaded
2024-12-12 15:14:57,121 - model_handler.py[line:502] - INFO: Model cabinet_meter Relased
2024-12-12 15:59:14,936 - model_handler.py[line:56] - INFO: Model lightning_rod_current_meter Loaded
2024-12-12 16:20:10,819 - model_handler.py[line:466] - INFO: Model cabinet_meter Loaded
2024-12-12 16:20:12,977 - model_handler.py[line:503] - INFO: Model cabinet_meter Relased
2024-12-12 17:41:51,601 - model_handler.py[line:39] - INFO: Model hat Loaded
2024-12-12 17:41:53,619 - model_handler.py[line:72] - INFO: Model hat Relased
2024-12-12 17:42:27,262 - model_handler.py[line:43] - INFO: Model intrusion Loaded
2024-12-12 17:42:35,702 - model_handler.py[line:74] - INFO: Model intrusion Relased
2024-12-12 17:43:57,751 - model_handler.py[line:793] - INFO: Model wandering Loaded
2024-12-12 17:44:00,499 - model_handler.py[line:825] - INFO: Model wandering Relased
2024-12-12 17:44:51,903 - model_handler.py[line:56] - INFO: Model lightning_rod_current_meter Loaded
2024-12-12 17:44:52,369 - model_handler.py[line:89] - INFO: Model lightning_rod_current_meter Relased
2024-12-12 17:45:13,586 - model_handler.py[line:466] - INFO: Model cabinet_meter Loaded
2024-12-12 17:45:15,686 - model_handler.py[line:503] - INFO: Model cabinet_meter Relased
2024-12-12 18:28:26,403 - model_handler.py[line:55] - INFO: Model hat Loaded on npu:0
2024-12-12 18:28:27,848 - model_handler.py[line:69] - INFO: Model hat Relased on npu:0
2024-12-12 18:30:19,779 - model_handler.py[line:58] - INFO: Model intrusion Loaded on npu:0
2024-12-12 18:30:19,788 - exceptions.py[line:34] - ERROR: Traceback (most recent call last):
  File "/home/HwHiAiUser/lingqi-xutao/gwproc/core/exceptions.py", line 18, in wrapper
    return func(*args, **kwargs)
  File "/home/HwHiAiUser/lingqi-xutao/gwproc/intrusion/model_handler.py", line 142, in inference
    input_name = sess.get_inputs()[0].name
AttributeError: 'AclLiteModel' object has no attribute 'get_inputs'

2024-12-12 18:30:20,873 - model_handler.py[line:71] - INFO: Model intrusion Relased on npu:0
2024-12-12 18:39:56,070 - model_handler.py[line:58] - INFO: Model intrusion Loaded on npu:0
2024-12-12 18:39:57,918 - model_handler.py[line:71] - INFO: Model intrusion Relased on npu:0
2024-12-12 18:40:46,922 - model_handler.py[line:43] - INFO: Model intrusion Loaded
2024-12-12 18:40:49,789 - model_handler.py[line:74] - INFO: Model intrusion Relased
2024-12-12 18:44:05,877 - model_handler.py[line:43] - INFO: Model intrusion Loaded
2024-12-12 18:44:07,323 - model_handler.py[line:55] - INFO: Model hat Loaded on npu:0
2024-12-12 18:44:08,015 - model_handler.py[line:69] - INFO: Model hat Relased on npu:0
2024-12-12 18:45:04,852 - model_handler.py[line:55] - INFO: Model hat Loaded on npu:0
2024-12-12 18:45:25,790 - model_handler.py[line:55] - INFO: Model hat Loaded on npu:0
2024-12-12 18:46:10,750 - model_handler.py[line:55] - INFO: Model hat Loaded on npu:0
2024-12-12 18:49:39,845 - model_handler.py[line:55] - INFO: Model hat Loaded on npu:0
2024-12-12 18:50:11,107 - model_handler.py[line:55] - INFO: Model hat Loaded on npu:0
2024-12-12 18:50:11,972 - model_handler.py[line:69] - INFO: Model hat Relased on npu:0
2024-12-12 18:51:25,175 - model_handler.py[line:58] - INFO: Model intrusion Loaded on npu:0
2024-12-12 18:51:26,978 - model_handler.py[line:71] - INFO: Model intrusion Relased on npu:0
2024-12-12 19:04:45,060 - model_handler.py[line:808] - INFO: Model wandering Loaded on npu:0
2024-12-12 19:04:45,516 - exceptions.py[line:34] - ERROR: Traceback (most recent call last):
  File "/home/HwHiAiUser/lingqi-xutao/gwproc/core/exceptions.py", line 18, in wrapper
    return func(*args, **kwargs)
  File "/home/HwHiAiUser/lingqi-xutao/gwproc/wandering/model_handler.py", line 875, in inference
    input_name = sess.get_inputs()[0].name
AttributeError: 'AclLiteModel' object has no attribute 'get_inputs'

2024-12-12 19:04:47,041 - model_handler.py[line:822] - INFO: Model wandering Relased on npu:0
2024-12-12 19:15:14,535 - model_handler.py[line:808] - INFO: Model wandering Loaded on npu:0
2024-12-12 19:15:14,703 - exceptions.py[line:34] - ERROR: Traceback (most recent call last):
  File "/home/HwHiAiUser/lingqi-xutao/gwproc/core/exceptions.py", line 18, in wrapper
    return func(*args, **kwargs)
  File "/home/HwHiAiUser/lingqi-xutao/gwproc/wandering/model_handler.py", line 875, in inference
    if self.platfrm == 'ONNX':
AttributeError: 'BehaviorDetectWanderingHandler' object has no attribute 'platfrm'

2024-12-12 19:15:16,088 - model_handler.py[line:822] - INFO: Model wandering Relased on npu:0
2024-12-12 19:16:28,365 - model_handler.py[line:808] - INFO: Model wandering Loaded on npu:0
2024-12-12 19:16:29,272 - exceptions.py[line:34] - ERROR: Traceback (most recent call last):
  File "/home/HwHiAiUser/lingqi-xutao/gwproc/core/exceptions.py", line 18, in wrapper
    return func(*args, **kwargs)
  File "/home/HwHiAiUser/lingqi-xutao/gwproc/wandering/model_handler.py", line 908, in inference
    output = sess.execute(im)[0]
  File "/usr/local/Ascend/ascend-toolkit/latest/thirdpart/python/acllite_model.py", line 270, in execute
    if size != self.input_buffer_sizes[i]:
IndexError: list index out of range

2024-12-12 19:16:30,156 - model_handler.py[line:822] - INFO: Model wandering Relased on npu:0
2024-12-12 19:20:46,081 - model_handler.py[line:808] - INFO: Model wandering Loaded on npu:0
2024-12-12 19:20:48,134 - model_handler.py[line:822] - INFO: Model wandering Relased on npu:0
2024-12-12 19:32:08,656 - model_handler.py[line:75] - INFO: Model lightning_rod_current_meter Loaded on npu:0
2024-12-12 19:32:09,220 - model_handler.py[line:86] - INFO: Model lightning_rod_current_meter Relased on npu:0
2024-12-12 19:39:47,560 - model_handler.py[line:489] - INFO: Model cabinet_meter Loaded on npu:0
2024-12-12 19:39:49,256 - model_handler.py[line:500] - INFO: Model cabinet_meter Relased on npu:0
2024-12-12 22:22:03,385 - model_handler.py[line:466] - INFO: Model cabinet_meter Loaded
2024-12-12 22:22:05,374 - model_handler.py[line:503] - INFO: Model cabinet_meter Relased
2024-12-12 22:28:29,146 - model_handler.py[line:466] - INFO: Model cabinet_meter Loaded
2024-12-12 22:28:31,149 - model_handler.py[line:503] - INFO: Model cabinet_meter Relased
2024-12-12 22:29:26,477 - model_handler.py[line:489] - INFO: Model cabinet_meter Loaded on npu:0
2024-12-12 22:29:27,124 - model_handler.py[line:500] - INFO: Model cabinet_meter Relased on npu:0
2024-12-12 22:31:02,594 - model_handler.py[line:489] - INFO: Model cabinet_meter Loaded on npu:0
2024-12-12 22:31:04,134 - model_handler.py[line:500] - INFO: Model cabinet_meter Relased on npu:0
2024-12-12 22:32:19,264 - model_handler.py[line:489] - INFO: Model cabinet_meter Loaded on npu:0
2024-12-12 22:32:20,088 - exceptions.py[line:34] - ERROR: Traceback (most recent call last):
  File "/home/HwHiAiUser/lingqi-xutao/gwproc/core/exceptions.py", line 18, in wrapper
    return func(*args, **kwargs)
  File "/home/HwHiAiUser/lingqi-xutao/gwproc/cabinet_meter/model_handler.py", line 634, in inference
    mask_maps = self.process_mask_output(box_out, mask_pred, output_pointer[1])
IndexError: index 1 is out of bounds for axis 0 with size 1

2024-12-12 22:32:21,141 - model_handler.py[line:500] - INFO: Model cabinet_meter Relased on npu:0
2024-12-12 22:35:14,061 - model_handler.py[line:489] - INFO: Model cabinet_meter Loaded on npu:0
2024-12-12 22:35:14,151 - exceptions.py[line:34] - ERROR: Traceback (most recent call last):
  File "/home/HwHiAiUser/lingqi-xutao/gwproc/core/exceptions.py", line 18, in wrapper
    return func(*args, **kwargs)
  File "/home/HwHiAiUser/lingqi-xutao/gwproc/cabinet_meter/model_handler.py", line 630, in inference
    print(f'3: output pointer type: {type(output_pointer)}, shape: {output_pointer.shape}')
AttributeError: 'list' object has no attribute 'shape'

2024-12-12 22:35:15,161 - model_handler.py[line:500] - INFO: Model cabinet_meter Relased on npu:0
2024-12-12 22:36:03,077 - model_handler.py[line:489] - INFO: Model cabinet_meter Loaded on npu:0
2024-12-12 22:36:06,166 - model_handler.py[line:500] - INFO: Model cabinet_meter Relased on npu:0
2024-12-13 09:53:23,785 - model_handler.py[line:489] - INFO: Model cabinet_meter Loaded on npu:0
2024-12-13 09:53:26,729 - model_handler.py[line:500] - INFO: Model cabinet_meter Relased on npu:0
